stable/openstack-telemetry/README.md

```diff
@@ -1,9 +1,308 @@
 
 Certain configuration options within the bundle may need to be adjusted prior to deployment to fit your particular set of hardware. For example, network device names and block device names can vary, and passwords should be yours.
 
-For full details on the base cloud deployment please refer to the [Basic OpenStack Cloud][] bundle.
+For example, a section similar to this exists in the bundle.yaml file.  The third "column" are the values to set.  Some servers may not have eno2, they may have something like eth2 or some other network device name.  This needs to be adjusted prior to deployment.  The same principle holds for osd-devices.  The third column is a whitelist of devices to use for Ceph OSDs.  Adjust accordingly by editing bundle.yaml before deployment.
+
+Servers should have:
+
+ - A minimum of 8GB of physical RAM.
+ - Enough CPU cores to support your capacity requirements.
+ - Two disks (identified by /dev/sda and /dev/sdb); the first is used by MAAS for the OS install, the second for Ceph storage.
+ - Two cabled network ports on eno1 and eno2 (see below).
+
+Servers should have two physical network ports cabled; the first is used for general communication between services in the Cloud, the second is used for 'public' network traffic to and from instances (North/South traffic) running within the Cloud.
+
+
+> **Important**: The three MAAS nodes are needed for the actual OpenStack cloud;
+  they do not include the Juju controller. You actually need a minimum of four 
+  nodes. The controller node however can be a smaller system (1 CPU and 4 GiB
+  memory). Juju [constraints][juju-constraints] (e.g. 'tags') can be used to
+  target this smaller system at controller-creation time.
+
+## Components
+
+ - 3 Nodes for Nova Compute and Ceph, with RabbitMQ, MySQL, Keystone, Glance, Neutron, OVN, Nova Cloud Controller, Ceph RADOS Gateway, Aodh, Gnocchi, Ceilometer, Memcached, Cinder and Horizon under LXC containers.
+
+All physical servers (not LXC containers) will also have NTP installed and configured to keep time in sync.
+
+## Download the bundle
+
+Modifications will typically need to be made to this bundle for it to work in
+your environment. You will also require access to a credentials/init file.
+
+The Charm Tools allow you to download charms and bundles from the Charm Store.
+Install them now and then download the bundle:
+
+    sudo snap install charm --classic
+    charm pull openstack-telemetry ~/openstack-telemetry
+
+## Network spaces and overlays
+
+If the MAAS cluster contains network spaces you will need to bind them to the
+applications to be deployed. One way of doing this is with the
+`openstack-telemetry-spaces-overlay.yaml` overlay bundle that ships with the bundle.
+
+Like the main bundle file, it will likely require tailoring. The file employs
+the variable method of assigning values. The actual space name should be the
```

Comment:
"uses YAML aliases" I would say. If I were a newbie to Juju and charms, I would be confused as to whether this is a Juju mechanism or something else. YAML aliases are not specific to Juju at all which is useful to clarify.